### DISCUSSION TITLE
Documentation entirely lacks system requirements section

### DISCUSSION THREAD

**Discussion by SeaMac** [mifi/lossless-cut#1744] [Ideas]
You need to have a section of documentation for people with older systems (who get left behind when you suddenly stop supporting their OS) listing which versions of Lossless Cut will work with which system Max.

Lossless Cut wants me to upgrade to an incompatible version - So many people are going to have it stop working suddenly because they still use a system LESS that Mac 15 ...

You should at least have the courtesy to notify people when the system requirements change - so they'll know FOR Sure you left them behind.

For instance:
https://github.com/mifi/lossless-cut/issues/1386
LC 3.47 is the last to support Mac up to 10.12
LC 3.56 is the last to support ANY Mac below Mac 15
And so on.

I can't upgrade my old system so I'm stuck on 3.56 - I'm SURE I won't donate any further time, or money, to this project: since I am no longer supported. 

Let People KNOW not to download an incompatible version, maybe?!?
Thank you for 3.56 It is my prime video editor and I'm glad you supported Older systems for so long. But you just halved your Mac active userbase - going forward - anyone Who COULDN'T upgrade to 64 bit only systems JUST Got Dropped.

**Comment by mifi** [comment]
I get that it’s frustrating to be left behind, but the truth is that if software were to support all old versions of operating systems, there wouldn’t be any progress, because developers would spend all our time testing and working around limitations in old platforms. The software world needs to be able to move forward. A 32 bit product that has been working for 16+ years, I would say it has served its purpose and reached an end of life..

if you purchased losslesscut on the mac app store, it will automatically stop auto updates from occurring for incompatible platforms. For github, there are so many different platforms and OSes that I dont want to spend my time creating logic for that. But yes of course we could make a page with list of compatibilities by version. Losslesscut is open source so you are also free to contribute a PR if you want to see this improved…

**Comment by mifi** [comment]
i've added a page: https://github.com/mifi/lossless-cut/blob/master/requirements.md

feel free to suggest changes.

**Comment by mattack1** [comment]
Since numbering schemes for different software varies, that info is useful but still hard to figure out.  For example:
v3.58.0 [dropped support] for MacOS 10.14 and older.

So in that case, if I needed to run on 10.14, I'm not sure if I have to download 3.57.0 (and if I am even still able to download the old versions).

Perhaps wording it the other way (I'll presume my version number guess is correct)

macOS 10.13 - 10.14 - last compatible version is v3.57.0 here [link provided]
etc...

**Reply by mifi** [reply]
that's a good idea! feel free to update: https://github.com/mifi/lossless-cut/edit/master/requirements.md
